internal/assert/assertions_test.go

```diff
@@ -1064,7 +1064,6 @@ Diff:
 @@ -1,2 +1,2 @@
 -(time.Time) 2020-09-24 00:00:00 +0000 UTC
 +(time.Time) 2020-09-25 00:00:00 +0000 UTC
- 
```

Comment:
This test appears to expect this blank like: https://parsley.mongodb.com/test/mongo_go_driver_v1_tests_42_plus_zlib_zstd_support__version~7.0_os_ssl_40~rhel87_64_go_1_20_test_replicaset_auth_nossl_patch_3a07cb190cee239b59795758a6f26898cb8cb78d_654105853e8e86b87f9395c1_23_10_31_13_47_51/0/125d94d8de41dfe15df278447d79190a?bookmarks=0,10711,24230&shareLine=10713

Reply:
Thanks, I'll play with the whitespace settings.